tmt/export.py

```diff
@@ -513,15 +604,173 @@ def export_to_nitrate(test):
 
     # Optionally link Bugzilla to Nitrate case
     if link_bugzilla and verifies_bug_ids:
+        bz_set_coverage(
+            verifies_bug_ids, int(nitrate_case.id), NITRATE_TRACKER_ID)
```

Comment:
Why is the the try-except block and dry mode handling removed? Originally it was:

    # Optionally link Bugzilla to Nitrate case
    if link_bugzilla and verifies_bug_ids:
        try:
            if not dry_mode:
                bz_set_coverage(
                    bz_instance, verifies_bug_ids, int(
                        nitrate_case.id))
            echo(style("Linked to Bugzilla: {}.".format(", ".join(
                [f"BZ#{bz_id}" for bz_id in verifies_bug_ids])), fg='magenta'))
        except Exception as err:
            raise ConvertError("Couldn't update bugs", original=err)

Probably a merge error?